### PR TITLE
[KIP-848] Run integration tests with both the "classic" and "consumer" consumer groups

### PR DESCRIPTION
**Pull request by emasab**

### CODE REVIEW

File: kafka/testresources/kraft/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM debian:bookworm
+
+RUN apt update
+RUN apt install -y git curl
+RUN apt install -y openjdk-17-jdk
+
+RUN git clone https://github.com/apache/kafka.git
```
Comment (milindl) [inline kafka/testresources/kraft/Dockerfile:7]: I think we should checkout to a branch or a tag after this for making sure tests don't break at some random point in the future

**Review by milindl — APPROVED** [review]
Looks okay except one comment

File: kafka/testresources/kraft/Dockerfile (hunk 1 above)
Comment (emasab) [reply kafka/testresources/kraft/Dockerfile:7]: At the moment there's no tag to checkout as 3.7.0 needed some fixes I could leave current commit in trunk and we update it when needed before 3.8.0

**Comment by emasab** [comment]
@milindl sent a commit, please check again

**Review by milindl — APPROVED** [review]
The PR itself looks OK barring the KRaft+classic failures

**Pull request merged**